src/main/java/fr/ospiea/oscra/employee/object/Employee.java

```diff
@@ -0,0 +1,113 @@
+// tag::sample[]
+package fr.ospiea.oscra.employee.object;
+
+import fr.ospiea.oscra.common.AbstractEntity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+public class Employee extends AbstractEntity implements Serializable{
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    @Enumerated(EnumType.STRING)
+    private EnabledType enabled;
```

Comment:
In fact, the "enabled" is required by using spring security form authentication.
Example:
 authbuilder.jdbcAuthentication()
                .dataSource(restDataSource)
                .passwordEncoder(passwordEncoder())
                .usersByUsernameQuery("select email,password,enabled from employee where email=?";)
                .authoritiesByUsernameQuery(getAuthoritiesQuery());

Here, the pre-defined method by spring security usersByUsernameQuery() should return 3 fields: username, password, enabled.